### DISCUSSION TITLE
The contents in the document are missing?!

### DISCUSSION THREAD

**Discussion by cylinch82** [lululxvi/deepxde#1218] [General]
I found that the API contents were all gone in the document. Does anyone has the same problem?

**Comment by lululxvi** [comment]
Umm, should be fixed at https://github.com/lululxvi/deepxde/pull/1209
@Saransh-cpp Could you check?

**Reply by Saransh-cpp** [reply]
Yes, on it, this is very weird.

**Reply by cylinch82** [reply]
It's back. Thanks
